src/lib.rs

```diff
@@ -22,12 +22,12 @@ mod duration;
 mod wrapper;
 mod date;
 
-pub use duration::{parse_duration, Error as DurationError};
-pub use duration::{format_duration, FormattedDuration};
-pub use wrapper::{Duration, Timestamp};
-pub use date::{parse_rfc3339, parse_rfc3339_weak, Error as TimestampError};
-pub use date::{
+pub use self::duration::{parse_duration, Error as DurationError};
```

Comment:
Is this self required? I think referring module next to declaring it `mod duration;`. Shouldn't have to

Reply:
I prefer to add `self` for exporting module in `lib.rs` to distinguish the exports of other libraries. I think this is more obvious, although `humantime` does not depend on other libraries by now...

Reply:
Well, I don't see this style used very often. So it's convenient for me to remove `self`. I don't think we would ever add a dependency.